x-pack/plugins/fleet/kibana.json

```diff
@@ -4,14 +4,13 @@
   "server": true,
   "ui": true,
   "configPath": ["xpack", "fleet"],
-  "requiredPlugins": ["licensing", "data"],
+  "requiredPlugins": ["licensing", "data", "encryptedSavedObjects"],
```

Comment:
__note:__ for 7.12 we decided to not disable plugin automatically and since [you still depend SO migrations](https://github.com/elastic/kibana/issues/81619#issuecomment-767173876) it makes sense to keep this as a required dependency for now. Let me know if you have any concerns.

Reply:
Make sense 👍 and we will fix our SO migration